dm-frontend/faultyDevice/faultdevice.html

```diff
@@ -0,0 +1,122 @@
+                        <!-- <span>                  
+                          <input type="text" class="serial-number" id="device_serial_number"  name="search"  serialNumber="" placeholder="enter device serial number...">
+                    </span> -->
+
+                    </div>
+                    <span  style="float: right; margin: 2%;"><input type='text' id="serial-number" data-find="" placeholder="enter serial number"
+                           ></span>
+                    <!-- <div>
+                          <select id="request-status">
+                            <option class="status" id="All">All</option>
+                            <option class="status"  id ="ret">Returned</option>
+                            <option class="status" id="rej">Rejected</option>
+                          </select>  style="display: flex; justify-content: center;"
+                    </div> -->
+                    <div class="user_data_table" >
+                        <table class="mdl-data-table mdl-js-data-table mdl-shadow--2dp">
+                            <thead id="tableHead" sort="" sortby="">
+                                <tr>
+                                    <th class="mdl-data-table__header--sorted-ascending" id="user">
+                                        User Name</th>
+
+                                    <th >
+                                        Device Type</th>
+
+                                    <th class="mdl-data-table__header--sorted-ascending"  id="device"> Device Name</th>
+
+                                    <th class="mdl-data-table__header--sorted-ascending" id="serialnumber">
+                                        Device Serial Number</th>
+
+                                    <th>Device Status</th>
+
+                                    <th>Complaint</th>
+
+                                    <th class="mdl-data-table__header--sorted-ascending" id="date">Complait Date</th>
+
+                                    <th id="admin">
+                                        Action</th>
+
+                                </tr>
+                            </thead>
+                            <tbody id="Request_data">
+
+                            </tbody>
+                        </table>
+                    </div>
+
+                </div>
+                <br />
+                <div class="paging" id="pagination" pageNum="1" result-count="0">
+
+                    <!-- <input type="submit" class="page" id="1" value="1" >
+                    <input type="submit" class="page" id="2" value="2" >
+                    <input type="submit" class="page" id="3" value="3" > -->
+
+                </div>
+            </div>
+        </main>
+    </div>
+    <div class="load" id="loading">
+        <div class= "loader"></div>
+    </div>
+
+
+
+    <script data-main="faultIndex.js" src="https://requirejs.org/docs/release/2.3.6/minified/require.js"></script>
```

Comment:
We no longer need require.js
Read [here](https://github.com/rishikant05/Device-Management/blob/develop/README.md)
Also add your file in webpack config